src/TestExplorer/TestRunner.ts

```diff
@@ -106,100 +108,207 @@ export class TestRunner {
             return;
         }
 
+        try {
+            if (shouldDebug) {
+                await this.debugSession(token);
+            } else {
+                await this.runSession(token);
+            }
+        } catch (error) {
+            const reason = error as string;
+            if (reason) {
+                this.testRun.appendOutput(reason.toString());
+            }
+            console.log(error);
+        }
+
+        this.testRun.end();
+    }
+
+    /**
+     * Edit launch configuration to run tests
+     * @param debugging Do we need this configuration for debugging
+     * @param outputFile Debug output file
+     * @returns
+     */
+    private createLaunchConfigurationForTesting(
+        debugging: boolean,
+        outputFile: string | null = null
+    ): vscode.DebugConfiguration | null {
+        const testList = this.testItems.map(item => item.id).join(",");
+
+        if (process.platform === "darwin") {
+            // if debugging on macOS need to create a custom profile so we can set the system
```

Comment:
I'm assuming this should be
```suggestion
            // if debugging on macOS need to create a custom procfile so we can set the system
```

Reply:
It's a custom launch configuration, I'll change from profile to that